src/state/user/hooks.tsx

```diff
@@ -158,9 +172,29 @@ export function useUserSlippageTolerance(): [number, (slippage: number) => void]
     [dispatch],
   )
 
+  const { isSwapPage } = usePageLocation()
+  if (useLocation && !isSwapPage) {
```

Comment:
should have isPoolPage.
```
if (useLocation && isPoolPage) {
```
because when we have another slippage setting (ex: campaign slippage, ... ) it will be easier for us to check.

Reply:
But the logic is "is swap page or not", there might be another page other than Swap or Pool page. All pages, that is not in Swap page, would use the same setting

Reply:
same here, the new logic is complicated and unneccesary